src/Data/Nat/Bounded/Base.agda

```diff
@@ -0,0 +1,96 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Bounded natural numbers, and their equivalence to `Fin`
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Nat.Bounded.Base where
+
+open import Data.Nat.Base as ℕ
+import Data.Nat.DivMod as ℕ
+import Data.Nat.Properties as ℕ
+open import Data.Fin.Base as Fin using (Fin)
+import Data.Fin.Properties as Fin
+open import Relation.Binary.Core using (Rel)
+open import Relation.Binary.Definitions
+open import Relation.Nullary.Decidable.Core using (recompute)
+open import Relation.Nullary.Negation.Core using (¬_)
+
+private
+  variable
+    m n o : ℕ
+
+------------------------------------------------------------------------
+-- Definition
+
+record ℕ< (n : ℕ) : Set where
+  constructor ⟦_⟧<_
+  field
+    ⟦_⟧ : ℕ
+    .bounded : ⟦_⟧ < n
```

Comment:
I think we should use `Data.Irrelevant` because we don't have irrelevant projects and so manipulating these
records is annoying in my experience.

Reply:
Well, in that case, it's probably better simply to define this as an instance of `Data.Refinement`? 
That being so, I'll reconvert to `DRAFT` and take it off the v2.2 table for the time being...